modules/tinymce/src/core/main/ts/api/dom/EventUtils.ts

```diff
@@ -108,9 +91,8 @@ const fix = function <T extends any> (originalEvent: T, data?): EventUtilsEvent<
     event.target = event.srcElement || document;
   }
 
-  // Experimental shadow dom support
-  if (Env.experimentalShadowDom) {
```

Comment:
This is technically a breaking change, so should we be doing it? I'm on the fence since it was behind an experimental flag.

Reply:
The change to composedPath should be fine - it changes from throwing to working properly. 

In terms of the target:
- if experimental flag is off - no change in behaviour
- if experiemental flag is on - we used to tweak the target, now we don't

So, basically, the flag and its effect have been removed. 

The new behaviour is more consistent with native events.

Anyone relying on the flag can switch from reading `target` to calling `getComposedPath`. So, anyone using an experiemental flag now has a workaround. 

Also, TinyMCE wasn't particularly useful in a shadow root before this change - even with that flag set.

I think the change is ok on balance. I would argue that it's ok to change the behaviour of an undocumented, experiemental flag in a minor release.

Reply:
Discussed offline with @lnewson, @EkimChau and @metricjs and we've concluded that the change is ok.